client/stylesheets/_toCat.scss

```diff
@@ -5110,7 +5111,12 @@ On click the images will be displayed at normal size to complete the effect
 /* View Feedback */
 
 .profileUserFeedbackView {
-    width: 60%;
+    width: 90%;
+}
+
+.profileUserFeedbackView h2 {
+    font-weight: 400 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces